chapters/Terminology.md

```diff
@@ -198,28 +211,38 @@ Merkle Trees
 
 : ::Merkle Tree::
 
-Sparse Merkle Tree
+Non-Repudiation
 
-: A ::Merkle Tree:: data structure where each data point included
-  at the leaf of the tree is indexed.
+: Non-Repudiation is a feature of DID methods that can clearly state that all data
+  is available to present one canonical history for a DID.
 
-  This data structure enables proofs of both inclusion and non-inclusion of data
-  at a given index. The instantiation in this specification has 2^256 leaves
-  that are indexed by the SHA256 hash of a **did:btc1** identifier. The data
-  attested to at the leaves of the tree is the ::BTC1 Update:: for that
-  **did:btc1** identifier that indexed to the leaf.
+  If some data is needed but not available, the DID method MUST NOT allow DID
+  resolution to complete. Any changes to the history, such as may occur if a website
+  edits a file, MUST be detected and disallowed. The ::Late Publishing:: problem
+  breaks Non-Repudiation.
 
-SMT
+Offline Creation
 
-: ::Sparse Merkle Tree::
+: Offline Creation refers to when a **did:btc1** identifier and corresponding
+  initial DID document are created without requiring network interactions.
 
-Spares Merkle Trees
+**did:btc1** supports offline creation in two modes:
 
-: ::Sparse Merkle Tree::
+* Key Pair Deterministic Creation; and
+* DID Document Initiated Creation.
```

Comment:
The CRUD chapter uses these subsection titles:

- From Deterministic Key Pair
- From External Intermediate DID Document

"Deterministic" aligns pretty well in both places. But the other mode omits the "External Intermediate" qualifier here. Consistency might be something worth addressing (or in another PR if changing the CRUD chapter).